notebooks/105-language-quantize-bert/105-language-quantize-bert.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    for input_layer in quantized_model.inputs:
</pre><p>does it still required to reshape model? probably we can convert model with preserving dynamic shapes and pass them only into benchmark app. As I can see, other part of notebook uses dynamic shapes (I checked nncf works)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>the problem is NNCF POT will not support a model with dynamic input shape, so I provide a extra parameter: "input=input_info" during model conversion. otherwise some error message will be in output.</p>

Reply:
I tested with 2.5, it supports

Reply:
fixed